pubspec.yaml

```diff
@@ -9,23 +9,31 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
+  cupertino_icons: ^1.0.2
   flutter:
     sdk: flutter
-  cupertino_icons: ^1.0.2
+  flutter_native_splash: ^2.2.17
```

Comment:
This should be a dev dependency, as we will not have use for it as a direct dependency after it has generate the splash screen

Reply:
Done